client.go

```diff
@@ -158,24 +158,11 @@ func (c *Client) SetToken(token string) *Client {
 	return c
 }
 
-// SetLinodeBusyRetry configures resty to retry specifically on "Linode busy." errors
-// The retry wait time is configured in SetPollDelay
-func (c *Client) SetLinodeBusyRetry() *Client {
-	c.resty.
-		SetRetryCount(1000).
-		SetRetryMaxWaitTime(30 * time.Second).
-		AddRetryCondition(
-			func(r *resty.Response, _ error) bool {
-				apiError, ok := r.Error().(*APIError)
-				linodeBusy := ok && apiError.Error() == "Linode busy."
-				retry := r.StatusCode() == http.StatusBadRequest && linodeBusy
-				if retry {
-					log.Printf("[INFO] Received error %s - Retrying", apiError)
-				}
-				return retry
-			},
-		)
-
+// SetRetries adds retry conditions for Linode Busy. errors and 429s.efault to the given duration.
+func (c *Client) SetRetries() *Client {
+	addRetryConditional(linodeBusyRetryCondition)
+	addRetryConditional(tooManyRequestsRetryCondition)
```

Comment:
Perhaps these 2 retry condition predicates should be added in `init`? As it stands now, these will be duplicated every time a `linodego.New()` is called.

Reply:
good catch. Maybe the array belongs on the client struct

Reply:
Personally I think these would be good as options in the constructor, but we don't really have a notion of a config object that could be passed in currently.

Reply:
If designed from the ground up, perhaps, but most configuration today is handled by method calls on the client